tcms/templates/include/bugs_table.html

```diff
@@ -0,0 +1,21 @@
+{% load i18n %}
+{% load static %}
+
+<div class="col-xs-12 col-sm-6 col-md-4">
+    <div class="card-pf card-pf-accented">
+        <h2 class="card-pf-title">
+            <span class="fa fa-bug"></span>
+            {{ heading }}
+        </h2>
+
+        <div class="card-pf-body">
+            <table class="table {{ class }}" data-show-hyperlinks="{% if show_hyperlinks %}true{% else %}false{% endif %}">
```

Comment:
show-hyperlinks attribute doesn't seem to be used. Is this a leftover or something else?

Reply:
leftover